mlrun/artifacts/base.py

```diff
@@ -315,9 +315,14 @@ def get_target_path(self):
 
     def get_store_url(self, with_tag=True, project=None):
         """get the artifact uri (store://..) with optional parameters"""
-        tag = self.metadata.tree if with_tag else None
+        # with_tag was using tree instead of tag before the refactor,
+        # so the naming is changed but the behavior is the same
```

Comment:
wth?! please change the comment to a todo to fix it
I wonder if not specifying the tree will produce a valid URL